### DISCUSSION TITLE
Quest 1 connection issue - don't have permission to contact the host

### DISCUSSION THREAD

**Discussion by JT-os** [mozilla/hubs#3213] [General Questions & Discussion] [answered]
I was testing the plain wide open space room with someone and they were stuck in "entering the room" state.  They have a Quest 1 and are getting the error that they don't have permission to contact the host.  

It is a Mozilla hosted Hubs room and the link was revokable so it had a query string on the end of the link with the access code.

After a few attempts they tried to even use the Firefox browser and that didn't change anything.  Troubleshooting page didn't change anything either.

Thanks in advance for any additional recommendations.

**Comment by robinkwilson** [comment, marked as answer]
We'd need to test this, can you make an issue in the Hubs repository (https://github.com/mozilla/hubs/issues) about a Quest 1 having issues entering a room? 

Then link the new issue here!

**Reply by JT-os** [reply]
I will contact the person this happened to and report back.  Thanks : )
